crates/uv/tests/it/pip_install.rs

```diff
@@ -6190,7 +6190,7 @@ fn already_installed_local_path_dependent() {
 
     ----- stderr -----
       × No solution found when resolving dependencies:
-      ╰─▶ Because first-local was not found in the provided package locations and second-local==0.1.0 depends on first-local, we can conclude that second-local==0.1.0 cannot be used.
+      ╰─▶ Because first-local was not found in the package registry and second-local==0.1.0 depends on first-local, we can conclude that second-local==0.1.0 cannot be used.
```

Comment:
Do you know why these changed?

Reply:
This path used to short-circuit with `ErrorKind::NoIndex`, but now since we're considering flat indexes (via `--find-links`), we end up getting a `UnavailablePackage::NotFound` rendering as 'not found in the package registry' instead.

Reply:
I think this wording is actually a bit confusing, this is implying we did a registry lookup (even though we supplied `--no-index`) as opposed to just looking at the directories/URLs provided by `--find-links`. Something to address for sure.